db/citations.yml

```diff
@@ -10092,3 +10092,42 @@ zhu;gca11:
   wwwemail: ''
   wwwpub: http://slapper.apam.columbia.edu/bib-eu9iifae/papers/12zhugca.pdf
   year: '2012'
+lo;nc20:
+  author:
+    - Lo, Sheng-Han
+    - Feng, Liang
+    - Tan, Kui
+    - Huang, Zhehao
+    - Yuan, Shuai
+    - Wang, Kun-Yu
+    - Li, Bing-Han
+    - Liu, Wan-Ling
+    - Day, Gregory S.
+    - Tao, Songsheng
+    - Yang, Chun-Chuen
+    - Luo, Tzuoo-Tsair
+    - Lin, Chia-Her
+    - Wang, Sue-Lein
+    - Billinge, Simon J. L.
+    - Lu, Kuang-Lieh
+    - Chabal, Yves J.
+    - Zou, Xiaodong
+    - Zhou, Hong-Cai
+  doi: 10.1038/s41557-019-0364-0
+  entrytype: article-journal
+  facility: bnl
+  grant: tamu17
+  journal: Nature Chemistry
+  nb: ''
```

Comment:
we need a "month" field with the month of publication.

``nb`` would be ``art`` for article and possibly ``hilite`` if it is a highlight paper.  I think we could make it a highlight since it is in Nature Chemistry, though against this is the fact that our contribution was small.

We are not heavily using the nb tags yet, but we will in the future when we build the whole web-page from here